misc/packaging/README.md

```diff
@@ -0,0 +1,117 @@
+# ubuntuSoS
+this scripts show how to build acrn-hypervisor and acrn-kernel deb package
+
+1. setup native ubuntu
+
+install python3
+
+config proxy
+
+sudo su
+
+
+2. config json file
+
+
+{
+
+	"//":"release ubuntu as sos verion",
+
+	"install_package":"true",---------->this 1st time should set to true if you did not install acrn related compile package, if already install,set to false
+
+	"gcc_version":"7.3.0",---------->higher than this
+
+	"binutils":"2.27",---------->higher than this
+
+	"//":"acrn-hypervisor config",
+
+	"build_acrn":"true",---------->if you need build acrn-hypervisor
+
+	"sync_acrn_code":"true",---------->if you need sync acrn-hypervisor code 
+
+	"acrn_repo":"https://github.com/projectacrn/acrn-hypervisor.git",---------->acrn repo
+
+	"release_version":"remotes/origin/release_2.0",---------->acrn release branch
+
+	"acrn_deb_package":"true",---------->if you need create acrn-hypervison deb
+
+	"install_acrn_deb":"false",---------->if you need install acrn-hypervisor deb
+
+	"build_cmd":---------->acrn-hypervisor build command should include scenario and board info and release type
+
+	{
+
+		"scenario":
+
+		{
+
+			"industry":"true",
+
+			"hybrid":"true",
+
+			"logical_partition":"true"
+
+
+		},
+
+		"board":
+
+		{
+
+			"nuc7i7dnb":"true",
+
+			"whl-ipc-i5":"true"
+
+		},
+
+		"release":"0"
+
+	},
+
+	"//":"kernel config",
+
+	"build_acrn_kernel":"true",---------->if you need build acrn-kernel
```

Comment:
same questions here, how do you know if you need to set these to "true" ?

Reply:
user should know whether to build or not